autorest/jsonrpc/__init__.py

```diff
@@ -111,7 +111,7 @@ def list_inputs(self) -> List[str]:
         """
 
     @abstractmethod
-    def get_value(self, key: str) -> str:
+    def get_value(self, key: str) -> Optional[Any]:
```

Comment:
I think Any includes Optional no? I'm surprised we could even write "Optional[Any]", I thought None was part of Any

Reply:
Yeah, it can just be Any, changing it to that now